src/sage/sets/recursively_enumerated_set.pyx

```diff
@@ -9,25 +9,23 @@ sets that are described by some ``seeds`` and a successor function
 graded, forest) or not. The elements of a set having a symmetric, graded or
 forest structure can be enumerated uniquely without keeping all of them in
 memory. Many kinds of iterators are provided in this module: depth first
-search, breadth first search or elements of given depth.
+search, breadth first search and elements of given depth.
 
 See :wikipedia:`Recursively_enumerable_set`.
 
-See documentation of :func:`RecursivelyEnumeratedSet` below for the
+See the documentation of :func:`RecursivelyEnumeratedSet` below for the
 description of the inputs.
 
 AUTHORS:
 
 - Sébastien Labbé, April 2014, at Sage Days 57, Cernay-la-ville
 
-EXAMPLES:
-
 No hypothesis on the structure
 ------------------------------
 
 What we mean by "no hypothesis" is that the set is not known
 to be a forest, symmetric, or graded. However, it may have other
-structure, like not containing an oriented cycle, that does not
+structures, such as not containing an oriented cycle, that do not
```

Comment:
No, it should be "structure, such as ... does" (indeed, I like "such as" better than "like")